### PR TITLE
Add displayBanner hook

### LINKED ISSUE #17606: displayBanner hook is missing
**Describe the bug**
displayBanner hook is missing in database and in hook.xml

**Additional information**
PrestaShop version: 1.7.*
PHP version: 7.2

### PR DESCRIPTION
**Pull request by jf-viguier**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  |  Add displayBanner hook in hooks.xml and upgrade sql
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/17606
| How to test?  | try to hook a widget module to displayBanner in positions page : the hook is missing. The hook is in the doc https://devdocs.prestashop.com/1.7/modules/concepts/hooks/list-of-hooks/ and in the header.tpl

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17607)

### CODE REVIEW

File: install-dev/data/xml/hook.xml [hunk 1]
```diff
@@ -101,6 +101,11 @@
       <title>Very top of pages</title>
       <description>Use this hook for advertisement or modals you want to load first</description>
     </hook>
+    <hook id="displayBanner">
+      <name>displayBanner</name>
```
Comment (PierreRambaud) [inline install-dev/data/xml/hook.xml:105]: This hook came from blockbanner module :thinking: 
I think this file must be updated but I don't know what is this file :sweat_smile: 
https://github.com/PrestaShop/PrestaShop/blob/develop/config/xml/themes/default.xml#L118

Comment (jf-viguier) [reply install-dev/data/xml/hook.xml:105]: The hook is launched in /themes/classic/templates/_partials/header.tpl
blockbanner uses this hook I guess ?

Comment (jf-viguier) [reply install-dev/data/xml/hook.xml:105]: default.xml file seems to define modules hooking at install ? If so, this file doesn't need to be changed.

Comment (PierreRambaud) [reply install-dev/data/xml/hook.xml:105]: But, the blockbanner doesn't exists anymore, it's ps_banner now :thinking:

Comment (micka-fdz) [reply install-dev/data/xml/hook.xml:105]: The default.xml file no longer seems to be used, all modules are for version 1.6 and this file is about old default-bootstrap theme.

Comment (PierreRambaud) [reply install-dev/data/xml/hook.xml:105]: It's used in `prestashop:update:configuration-file-hooks-listing` Symfony command

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by jf-viguier** [comment]
Thanks. I've put the sql part in 1.7.8 sql file. I don't know if it's ok or if there is a develop sql file for that.

**Comment by SD1982** [comment]
This pull request correctly fix #17606 . Thanks @jf-viguier !

**Comment by Progi1984** [comment]
Thanks @jf-viguier

**Pull request merged**